ocs_ci/ocs/resources/mcg.py

```diff
@@ -48,38 +50,31 @@ def __init__(self):
             aws_secret_access_key=self.access_key
         )
 
-    def s3_create_bucket(self, bucketname, region=config.ENV_DATA['region']):
+    def s3_list_all_bucket_names(self):
         """
-        Args:
-            bucketname: Name of the bucket to be created
-            region: Name of the region for the bucket to be created on
-
         Returns:
-            s3.Bucket object
+            list: A list of all bucket names
 
         """
-        return self.s3_resource.create_bucket(
-            Bucket=bucketname,
-            CreateBucketConfiguration={
-                'LocationConstraint': region
-            }
-        )
+        return [bucket.name for bucket in self.s3_resource.buckets.all()]
 
-    def s3_delete_bucket(self, bucket):
+    def oc_list_all_bucket_names(self):
         """
-        Args:
-            bucket: The bucket object to be deleted
+        Returns:
+            list: A list of all bucket names
 
         """
-        bucket.delete()
+        return [bucket.get('spec').get('bucketName')
+                for bucket
+                in OCP(namespace='openshift-storage', kind='obc').get().get('items')]
```

Comment:
Please optimize this call.
OCP(namespace='openshift-storage', kind='obc').get().get('items')] should be called once (1 call to api) and list comprehension on the returned list